xarray/tests/test_dataarray.py

```diff
@@ -3374,6 +3374,18 @@ def test_to_and_from_series(self):
         expected_da = self.dv.rename(None)
         assert_identical(expected_da, DataArray.from_series(actual).drop(["x", "y"]))
 
+    def test_from_series_sparse(self):
+        sparse = pytest.importorskip("sparse")
+
+        series = pd.Series([1, 2], index=[("a", 1), ("b", 2)])
+
+        actual_sparse = DataArray.from_series(series, sparse=True)
+        actual_dense = DataArray.from_series(series, sparse=True)
```

Comment:
isn't it sparse=False?

Reply:
Also it could use testing the output DataArray name when the input Series has and hasn't got a name.

Reply:
Oops, fixed.

There's already a case that checks `name=None` in the test above.